src/App/Resources/AppResources.resx

```diff
@@ -59,8 +59,9 @@
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
```

Comment:
Not sure why this formatting all changed. I guess something Rider did? Does Rider have a resx editor or did you have to modify this file manually?

Reply:
Rider did do this, but I believe its because it exceeded the line length.